blog/2021-12-20-openhab-3-1-release.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: BlogPost
+category: blog
+title: openHAB 3.2 Release
+author: Kai Kreuzer
+date: '2021-12-20T00:00:00+01:00'
+previewimage: /uploads/2021-12-20-header.jpg
+tags:
+  - news
+excerpt: >-
+  Our winter release openHAB 3.2 is out! It improves many details over 3.1 and again comes with a huge list of new add-ons.
+---
+Our winter release openHAB 3.2 is out! It improves many details over 3.1 and again comes with a huge list of new add-ons.
+
+<!-- more -->
+
+INTRO PLACEHOLDER
+
+## Rule Engine Enhancements 
+
+In the very early days of openHAB, the rules “DSL” was the only standard for writing custom rules.  This Java-like scripting language is based on Xtend and is still used widely today due to its stability and powerful built-in functionality.  Over time however,  the underlying rule infrastructure has matured allowing additional engines to coexist to provide a true polyglot scripting environment.  OpenHAB 3.2 ships with multiple rules engines including [Jruby](https://next.openhab.org/addons/automation/jruby/), [Jython](https://www.openhab.org/addons/automation/jythonscripting/), [Groovy](https://www.openhab.org/addons/automation/groovyscripting/), and a ES6 2021 compatible Javascript environment, [JS Scripting](https://www.openhab.org/addons/automation/jsscripting/). 
```

Comment:
I have a comment about capitalizations as usual 🙂 

```suggestion
In the very early days of openHAB, the rules “DSL” was the only standard for writing custom rules.  This Java-like scripting language is based on Xtend and is still used widely today due to its stability and powerful built-in functionality.  Over time however,  the underlying rule infrastructure has matured allowing additional engines to coexist to provide a true polyglot scripting environment.  openHAB 3.2 ships with multiple rules engines including [JRuby](https://next.openhab.org/addons/automation/jruby/), [Jython](https://www.openhab.org/addons/automation/jythonscripting/), [Groovy](https://www.openhab.org/addons/automation/groovyscripting/), and a ES6 2021 compatible Javascript environment, [JS Scripting](https://www.openhab.org/addons/automation/jsscripting/). 
```

Reply:
And maybe the double spaces could be fixed at the same time. 😎

Reply:
And `https://next.openhab.org/addons/automation/jruby/` should be `https://www.openhab.org/addons/automation/jruby/`.

Reply:
And according to my English lectures at school, it should be `an ES6` not `a ES6`. But I don't dare to tell this to a native speaker. 😆

Reply:
Thanks

Reply:
> And https://next.openhab.org/addons/automation/jruby/ should be https://www.openhab.org/addons/automation/jruby/

It should be:

https://www.openhab.org/addons/automation/jrubyscripting/